samples/extensions/shader_object/shader_object.cpp

```diff
@@ -171,10 +172,6 @@ bool ShaderObject::prepare(const vkb::ApplicationOptions &options)
 	camera.set_rotation({19.f, 312.f, 0.f});
 	camera.set_perspective(60.f, static_cast<float>(width) / static_cast<float>(height), 1024.f, 0.1f);
 
-	// For UI need to setup FBO and render pass
-	setup_framebuffer();
-	setup_render_pass();
```

Comment:
Are these calls no longer needed?

Reply:
They are called elsewhere, I believe this just removes the second call.

Reply:
Exactly. The calls in this spot were not only unnecessary, but also resulted in resources getting acquired multiple times without getting properly freed.